app/src/main/java/cz/muni/fi/pv256/movio2/uco_410434/ui/FilmListFragment.java

```diff
@@ -0,0 +1,185 @@
+package cz.muni.fi.pv256.movio2.uco_410434.ui;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.AppCompatRatingBar;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import cz.muni.fi.pv256.movio2.uco_410434.R;
+import cz.muni.fi.pv256.movio2.uco_410434.model.Category;
+import cz.muni.fi.pv256.movio2.uco_410434.model.EmptyListItem;
+import cz.muni.fi.pv256.movio2.uco_410434.model.Film;
+import cz.muni.fi.pv256.movio2.uco_410434.model.FilmsListItem;
+
+public class FilmListFragment extends Fragment {
+
+    private static final String TAG = FilmListFragment.class.getSimpleName();
+
+    private Context context;
+    private OnFilmSelectListener filmSelectListener;
+    private ArrayList<FilmsListItem> filmList;
+    private RecyclerView filmListView;
+
+    {
+        filmList = new ArrayList<>();
+        filmList.add(new Category("This week opening"));
+        filmList.add(new Film("Boss Baby", 3.5f, R.mipmap.bossbaby));
+        filmList.add(new Film("Sherlock ", 4.5f, R.mipmap.sherlock));
+        filmList.add(new Category("Old Classics"));
+        filmList.add(new Film("The Shining", 4.8f, R.mipmap.shining));
+        filmList.add(new Film("Casablanca", 3.4f, R.mipmap.casablanca));
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        context = getActivity().getApplicationContext();
+    }
+
+
+    @Override
+    public void onAttach(Context activity) {
+        super.onAttach(activity);
+
+        try {
+            filmSelectListener = (OnFilmSelectListener) activity;
```

Comment:
hmm, asi by som skor pridala ten listener pomocou nejakej metody (napr. v newInstance()) ..tu je celkom naviazany na tu aktivitu a pritom listener nemusi byt nutne implementovany v aktivite...
to si sa niekde inspiroval tymto riesenim? nehovorim ze to je zle, len sa mi to nezda.